src/Illuminate/Console/Scheduling/CallbackEvent.php

```diff
@@ -67,6 +69,23 @@ public function run(Container $container)
     }
 
     /**
+     * Write the given response to the disk.
+     *
+     * @param  string  $response
+     * @return void
+     */
+    protected function writeResponse($response)
+    {
+        if (! is_string($response)) {
+            return;
+        }
+
+        $openMode = $this->shouldAppendOutput ? 'a' : 'w';
+        $writeFile = new SplFileObject($this->output, $openMode);
+        $writeFile->fwrite($response);
```

Comment:
Writing `(new SplFileObject($this->output, $openMode))->fwrite($response)` would prevent the object hanging around I think.